### ISSUE TITLE
Setting GridSplitter Orientation property in XAML throws System.ArgumentException

### ISSUE THREAD

**Issue by wieslawsoltes** [Perspex/Perspex#376]
This:
```XAML
<GridSplitter Orientation="Horizontal"/>
```
and this:
```XAML
<GridSplitter Orientation="Vertical"/>
```
throws:
```
Error loading XAML: System.ArgumentException: Invalid value for Property 'Orientation': 'Horizontal' (System.String)
   w Perspex.PerspexObject.SetValue(PerspexProperty property, Object value, BindingPriority priority)
   w Perspex.Markup.Xaml.Context.PerspexXamlMemberValuePlugin.HandlePerspexProperty(Object instance, Object value)
   w Perspex.Markup.Xaml.Context.PerspexXamlMemberValuePlugin.SetValue(Object instance, Object value)
   w OmniXaml.ObjectAssembler.StateCommuter.AssignChildToParentProperty()
   w OmniXaml.ObjectAssembler.StateCommuter.AssociateCurrentInstanceToParent()
   w OmniXaml.ObjectAssembler.Commands.ValueCommand.Execute()
   w OmniXaml.ObjectAssembler.ObjectAssembler.Process(XamlInstruction instruction)
   w OmniXaml.TemplateHostingObjectAssembler.Process(XamlInstruction instruction)
   w Perspex.Markup.Xaml.Context.PerspexObjectAssembler.Process(XamlInstruction node)
   w OmniXaml.XamlXmlParser.Parse(IEnumerable`1 xamlNodes)
   w OmniXaml.XamlXmlParser.Parse(IXmlReader stream)
   w OmniXaml.XamlXmlLoader.Load(Stream stream, IXamlParser parser)
```

**Comment by wieslawsoltes** [comment]
Looks like `GridSplitter` is using nullable Orientation:
https://github.com/Perspex/Perspex/blob/master/src/Perspex.Controls/GridSplitter.cs#L26
and `StackPanel` not:
https://github.com/Perspex/Perspex/blob/master/src/Perspex.Controls/StackPanel.cs#L39

**Comment by wieslawsoltes** [comment]
I should add that setting `Ortientation` using styles works:

``` XAML
        <Style Selector="GridSplitter.horizontal">
            <Setter Property="Background" Value="Transparent"/>
            <Setter Property="Orientation" Value="Horizontal"/>
        </Style>
        <Style Selector="GridSplitter.vertical">
            <Setter Property="Background" Value="Transparent"/>
            <Setter Property="Orientation" Value="Vertical"/>
        </Style>
```

``` XAML
<GridSplitter Classes="horizontal" Grid.Row="1" Height="6"/>
```

``` XAML
<GridSplitter Classes="vertical" Grid.Column="1" Width="10"/>
```

**Comment by wieslawsoltes** [comment]
Actually using styles does not work I think:

``` XAML
        <Style Selector="GridSplitter.horizontal">
            <Setter Property="Background" Value="Red"/>
            <Setter Property="Orientation" Value="Vertical"/>
        </Style>
        <Style Selector="GridSplitter.vertical">
            <Setter Property="Background" Value="Green"/>
            <Setter Property="Orientation" Value="Horizontal"/>
        </Style>
```

![przechwytywanie](https://cloud.githubusercontent.com/assets/2297442/11986876/3d6633e0-a9d5-11e5-8b98-bcbcbb35a847.PNG)

**Comment by grokys** [comment]
https://github.com/SuperJMN/OmniXAML/issues/69

@susloparovdenis until this gets fixed, I'm going to make `GridSplitter.Orientation` non-nullable.

**Comment by grokys** [comment]
Fixed by 7ab944c0a5b7db49d605706cba9dd0b924c33978

**Comment by grokys** [comment]
It seems that the last NuGet package didn't have the Orientation fix as CI was failing. Should be fixed now.
